contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -0,0 +1,330 @@
+/**
+ *  @title Multiple Arbitrable Transaction
+ *  Bug Bounties: This code hasn't undertaken a bug bounty program yet.
+ */
+
+
+pragma solidity ^0.4.15;
+import "./Arbitrator.sol";
+
+/** @title Multiple Arbitrable Transaction
+ *  This is a a contract for multiple arbitrated transactions which can be reversed by an arbitrator.
+ *  This can be used for buying goods, services and for paying freelancers.
+ *  Parties are identified as "seller" and "buyer".
+ */
+ contract MultipleArbitrableTransaction {
+    string constant RULING_OPTIONS = "Reimburse buyer;Pay seller";
+    
+    struct Transaction {
+        address seller;
+        address buyer;
+        uint256 amount;
+        uint256 timeout; // Time in seconds a party can take before being considered unresponding and lose the dispute.
+        uint disputeId;
+        bool disputed;
+        Arbitrator arbitrator;
+        bytes arbitratorExtraData;
+        uint sellerFee; // Total fees paid by the seller.
+        uint buyerFee; // Total fees paid by the buyer.
+        uint lastInteraction; // Last interaction for the dispute procedure.
+        Status status;
+    }
+
+    Transaction[] transactions;
+
+    mapping (uint => uint) disputeTxMap;
```

Comment:
Dispute IDs can be duplicated across arbitrators so this won't work.

Reply:
hmm, noted. Let me see if I can find a workaround.